src/iceberg/schema_field.h

```diff
@@ -86,4 +86,16 @@ class ICEBERG_EXPORT SchemaField : public iceberg::util::Formattable {
   bool optional_;
 };
 
+template <typename T, typename... Args>
+inline SchemaField MakeRequiredField(int id, std::string name, Args&&... args) {
```

Comment:
```suggestion
inline SchemaField MakeRequiredField(int32_t id, std::string name, Args&&... args) {
```

Reply:
I've deleted these helpers because it doesn't fit well with the new added doc field.